docs/user-guide/managing-deployments.md

```diff
@@ -411,7 +411,9 @@ You can also run the [update demo](update-demo/) to see a visual representation
 
 ## In-place updates of resources
 
-Sometimes it’s necessary to make narrow, non-disruptive updates to resources you’ve created. For instance, you might want to add an [annotation](annotations.md) with a description of your object. That’s easiest to do with `kubectl patch`:
+Sometimes it’s necessary to make narrow, non-disruptive updates to resources you’ve created. For instance, you might want to add an [annotation](annotations.md) with a description of your object. 
+
+One way to do that is with `kubectl patch`:
```

Comment:
Can we patch something else, such as image? It makes more sense to use `kubectl annotate` rather than `patch` to update annotations.